@types/web-crypto.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module "crypto" {
+  namespace webcrypto {
+    const subtle: SubtleCrypto;
```

Comment:
Might not need with latest 16 or 18

Reply:
Seems like we already have not that outdated. Will check other PRs